mne/preprocessing/ica.py

```diff
@@ -243,7 +247,7 @@ class ICA(ContainsMixin):
 
     References
     ----------
-    .. [1] Hyvarinen, A., 1999. Fast and robust fixed-point algorithms for
+    .. [1] Hyvärinen, A., 1999. Fast and robust fixed-point algorithms for
```

Comment:
This might necessitate using `u"""` above

Reply:
😡 Python 2 💣